backend/app/api/api_v1/endpoints/auth.py

```diff
@@ -24,6 +27,13 @@ async def token(response: Response, code: str) -> AccessTokenOut:
     if "error" in data:
         raise HTTPException(status_code=status.HTTP_401_UNAUTHORIZED, detail=data)
 
+    user: User = user_utils.get_user(int(data["github_id"]))
```

Comment:
`get_user` causes validation errors if the user attempts to log in for the first time.